lib/monkey-mailer/adapters/mandrilapi.rb

```diff
@@ -59,6 +56,27 @@ def send_email(email)
         raise MonkeyMailer::DeliverError.new("Mandril response.code not equal to 200") unless response.code.to_i == 200
         puts "Response #{response.code} #{response.message}: #{response.body}"
       end
+
+      def parse_recipients(recipients_list, names_list = "", type = :to)
```

Comment:
Yes, I think we can safely remove it. It can be added later if support for other types is added.